Source/Common/MultiplayerServer.cs

```diff
@@ -82,6 +84,8 @@ public MultiplayerServer(ServerSettings settings)
 
             if (settings.lanAddress != null)
                 lanManager = new NetManager(new MpNetListener(this, false));
+
+        autosaveCountdown = settings.autosaveInterval * 2500 * 24;
```

Comment:
This indentation doesn't look right, but it might be github's fault.

Reply:
Ah, that must've slipped in when I changed from tabs to spaces. It's fixed now.